app/views/casework/crime_applications/_overview.html.erb

```diff
@@ -10,15 +10,26 @@
     </div>
 
     <% if overview.cifc? %>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          <%= label_text(:pre_cifc_reason) %>
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <span id="pre_cifc_reason">
+            <%= overview.pre_cifc_reason.presence || t(:not_provided, scope: 'values') %>
+          </span>
+        </dd>
+      </div>
```

Comment:
Will this break for cifc applications that have already been submitted that did not submit a reason?

Reply:
I tested this locally and it will display `Not provided`. I assume we wouldn't show the row.. Content and design aren't around this week so we won't be able to confirm this week but maybe we can not show the row for now and leave a comment on the ticket for them to confirm when they are back

Reply:
Happy to leave it as Not Provided - I'll double check if there have been any and if this is a moot point anyway!

Reply:
Yeah, there are 0 - noone likes your feature @mseedat-moj 😂

Reply:
Ok happy to approve in that case